packages/cli/src/cmds/validator/keymanager/decryptKeystoreDefinitions/worker.ts

```diff
@@ -0,0 +1,26 @@
+import fs from "node:fs";
+import {expose} from "@chainsafe/threads/worker";
+import {Transfer, TransferDescriptor} from "@chainsafe/threads";
+import {Keystore} from "@chainsafe/bls-keystore";
+import {LocalKeystoreDefinition} from "../interface.js";
+import {DecryptKeystoreWorkerAPI} from "./types.js";
+
+/**
+ * Decrypt a single keystore definition, returning the secret key as a Uint8Array
+ *
+ * NOTE: This is memory-intensive process, since decrypting the keystore involves running a key derivation function (either pbkdf2 or scrypt)
```

Comment:
this note is correct but based on the performance tests it looks like CPU is rather the bottleneck that prevents us from increasing the job concurrency to further speed up the key import

With job concurrency of 1

![image](https://user-images.githubusercontent.com/38436224/233351196-44ed93fc-81a4-4ee1-9aba-b283bae0dd10.png)